classes/cluster/mk22_lab_basic/fuel/config.yml

```diff
@@ -3,28 +3,24 @@ classes:
 - system.linux.system.single
 - system.linux.system.repo.tcp_salt
 - system.openssh.client.lab
-- system.salt.master.pkg
+- system.salt.master.git
```

Comment:
@elemoine @cznewt 
We have to decide: are we really going to change that? In mk22_lab_advanced the same changes required.
I assume that we should have 2 sets of model, prod which is for production, based on packages and development, git-based, for a quick updates and testing of not released features.